src/PolyhedralGeometry/PolyhedralFan/constructors.jl

```diff
@@ -4,6 +4,12 @@
 ###############################################################################
 ###############################################################################
 
+# We introduce this abstract (hidden) type to allow for other objects to be
+# used like polyhedral fans without duplicating too much code, concretely we
+# want to be able to directly access rays, maximal_cones, etc for
+# NormalToricVariety's.
+abstract type _FanLikeType end
+
```

Comment:
This is a minor. But maybe it would be clearer to write "objects of datatype `NormalToricVariety`"?